index.js

```diff
@@ -114,6 +115,10 @@ function session(options) {
   // get the cookie signing secret
   var secret = opts.secret
 
+  // should the header be set as token instead of cookie
+  var shouldReplaceCookieWithToken = opts.shouldReplaceCookieWithToken
+
```

Comment:
It looks like you have an extra carriage return here, other than that it looks good to me 👍🏼